pkg/controller/operators/olm/apiservices.go

```diff
@@ -263,9 +321,9 @@ func (a *Operator) installAPIServiceRequirements(desc v1alpha1.APIServiceDescrip
 	existingService, err := a.lister.CoreV1().ServiceLister().Services(csv.GetNamespace()).Get(service.GetName())
 	if err == nil {
 		// Check if the only owners are this CSV or in this CSV's replacement chain
-		if !ownerutil.Adoptable(csv, existingService.GetOwnerReferences()) {
-			return nil, fmt.Errorf("service %s not safe to replace: extraneous ownerreferences found", service.GetName())
-		}
+		// if ownerutil.Adoptable(csv, existingService.GetOwnerReferences()) {
+		// 	return nil, fmt.Errorf("service %s not safe to replace: extraneous ownerreferences found", service.GetName())
+		// }
 
 		// Append previous ownerrefs to the new Service
 		service.SetOwnerReferences(append(service.GetOwnerReferences(), existingService.GetOwnerReferences()...))
```

Comment:
Is stealing a service from an existing extension api-server deployment bad?. If it's managed by another CSV it will be recreated anyway.